### PR TITLE
Added env var CLIENT_MAX_BODY_SIZE

### PR DESCRIPTION
**Pull request by ebreton**
allow to override client_max_body_size in nginx conf.

fix typo in default template also (missing ‘;’)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,13 @@
-FROM nginx:1.11-alpine
+FROM nginx:1.11
```
Comment (MrReasonable) [inline Dockerfile]: Why did you remove the alpine part of this line?

File: global/wordpress.conf [hunk 2]
```diff
@@ -8,7 +8,8 @@ location / {
 }
 
 # Add trailing slash to */wp-admin requests.
-rewrite /wp-admin$ $scheme://$host$uri/ permanent;
+# EMB: unabled because of infinite loop
+# rewrite /wp-admin$ $scheme://$host$uri/ permanent;
```
Comment (MrReasonable) [inline global/wordpress.conf]: What URL are you going to that causes an infinite loop?

**Comment by ebreton** [comment]
Hi Ian, 

Those two last additions came after the pull request. They were not intended for release. but here are a bit of context:

1/ removed -alpine : to adjust to my baseline (all my images are based on nginx1-11, not -alpine. That is purely based on my choice and should not interfere with your strategy. Tags could help though ^^

2/ infinite loop: that comes on my context because of an haproxy in front of my wordresses that takes care of https. With default redirection from http to https. hence the loop (internet:https) -> haproxy (https->proxied_http) -> nginx4wp (redirect http) -> haproxy (http->https -> proxied_http) -> and so on

**Comment by MrReasonable** [comment]
Agreed about the tagging, I'll look into it later.  I will be happy to accept your Pull Request if you could remove those two unintentional parts of the PR.  Sorry I didn't merge the PR earlier, it slipped my mind.

**Comment by ebreton** [comment]
done. Thanks for the questions: nice discussion :)

**Review by MrReasonable — APPROVED** [review]
